howso/engine/trainee.py

```diff
@@ -1438,36 +1508,52 @@ def react(
                 When true outputs feature prediction stats for all (context
                 and action) features locally around the prediction. The stats
                 returned  are ("r2", "rmse", "spearman_coeff", "precision",
-                "recall", "accuracy", "mcc"). Confusion matrices may also be
-                returned by setting 'confusion_matrices' to true. Uses only the
-                context features of the reacted case to determine that area.
-                Relies on 'robust_residuals' flag.
-            - confusion_matrices : bool, optional
-                When true, will automatically set 'prediction_stats' to true and
-                return the confusion matrices alongside all of the other prediction
-                stats from 'prediction_stats' for all (context and action) features
-                locally around the prediction. Uses only the context features of
-                the reacted case to determine that area. Relies on 'robust_residuals'
-                flag.
+                "recall", "accuracy", "mcc", "confusion_matrix", "missing_value_accuracy").
+                Uses only the context features of the reacted case to determine that area.
+                Uses full calculations, which uses leave-one-out context features for
+                computations.
+            - prediction_stats_robust : bool, optional
+                When true outputs feature prediction stats for all (context
+                and action) features locally around the prediction. The stats
+                returned  are ("r2", "rmse", "spearman_coeff", "precision",
+                "recall", "accuracy", "mcc", "confusion_matrix", "missing_value_accuracy").
+                Uses only the context features of the reacted case to determine that area.
+                Uses robust calculations, which uses uniform sampling from the power set
+                of features as the contexts for predictions.
+            - selected_prediction_stats : list, optional. List of stats to output. When unspecified,
+                returns all except the confusion matrix. Allowed values:
+
+                - all : Returns all the the available prediction stats, including the confusion matrix.
```

Comment:
I would double check this list will render correctly in the documentation. I think you need to align subsequent lines for each bullet point with the start of the text.
i.e.
```
- confusion_matrix : A sparse map of actual feature value to a map of
  predicted feature value to counts.
```